linode/manager
# hotfix: Fix Backups Drawer, Region Select Order, Cloud Manager `v1.98.1`

## Description 📝

- Fix Backups Drawer by manually copying relevant changes from https://github.com/linode/manager/pull/9421
- Fixes Region Select Order

## Preview 📷

| Before  | After   |
| ------- | ------- |
| ![Screenshot 2023-07-28 at 1 53 37 PM](https://github.com/linode/manager/assets/115251059/5c3d5ba9-4a46-4304-ab2f-b54d32890237) | ![Screenshot 2023-07-28 at 1 53 10 PM](https://github.com/linode/manager/assets/115251059/120ae512-86fd-4e66-b98a-4f179484f98a) |
| <img width="517" alt="Screenshot 2023-07-28 at 1 58 13 PM" src="https://github.com/linode/manager/assets/125309814/af9e3faf-4cff-4ba0-a55e-bd4b389271d4">|<img width="511" alt="Screenshot 2023-07-28 at 1 58 31 PM" src="https://github.com/linode/manager/assets/125309814/8236f6c0-a35f-40b3-863b-d6233d07a5b5"> |

## How to test 🧪
- Test the backups drawer
  - You need some linodes on your account without backups 
  - If you don't see the backups drawer banner, go to `http://localhost:3000/profile/settings?preferenceEditor=true` and clear your preferences by setting the value to `{}`
- Verify region select order is ...
